index.html

```diff
@@ -31,38 +31,42 @@ <h1 class="textdeco1 text1width">José Fernández-Caminero</h1>
     
             <p class="crazyParagraph">I love Full-Stack Engineering and I can code websites that are both the most complex and the most efficient.</p>
     
-            <button class="button1 width90">Start collaboration</button>
+            <button class="button1 width90 bigButton1">Start collaboration</button>
```

Comment:
- [ ] Great job working on the design but I have small note in the header section so i recommend to add more spacing here so kindly add some margin/padding to not be very close to the image it is very important to make your project identical to the figma as much as possible since it is one of the requirements ♻🚀🔥💪🏻😊📌

- Your design
![image](https://user-images.githubusercontent.com/58553711/201233455-f32ce2ab-59f6-4fe4-a0bb-bd80fdf279e4.png)

- Figma 
![image](https://user-images.githubusercontent.com/58553711/201233495-3368a1cf-471a-4d5a-a727-2487e802b1a6.png)